Automattic/pocket-casts-android
# Add ability to hide custom media actions on media notification

## Description
This is a small PR that adds the ability to hide custom media actions (available on the Android 13 media style notification and Android Auto).
See comment https://github.com/Automattic/pocket-casts-android/issues/590#issuecomment-1354402436

## Testing Instructions

1. Play an episode and check that it displays two custom actions at the bottom right of the media notification.
2. Go To Profile > Settings > General > Defaults.
3. Click on "Media notification controls" under `Defaults` settings.
4. ✅ Notice a new settings row titled "Hide custom media actions".
5. Turn "Hide custom media actions" settings on.
6. ✅ Notice that "Prioritize ..." section is disabled and the two custom actions are hidden in the media notification.
7. Turn "Hide custom media actions" settings off.
8. ✅ Notice that the two custom actions are shown in the media notification.
9. Rearrange custom media actions from "Prioritize ..." section.
10. ✅ Notice that custom actions are rearranged in the media notification.

** Since these actions are also shown on Android Auto, they're also controlled by this new setting.
You can setup an `Android Auto` emulator https://developer.android.com/training/cars/testing#test-auto to test it. I've also included screenshots below from my testing.

## Screenshots or Screencast 

https://user-images.githubusercontent.com/1405144/213417484-b00afe83-80c3-449b-89fa-da3643125a95.mp4

Android Auto 

Shown | Hidden
---|---
<img width="796" alt="Screenshot 2023-01-19 at 3 47 27 PM" src="https://user-images.githubusercontent.com/1405144/213419671-d080d3dd-314a-4aaa-ad1c-362438187372.png"> |  <img width="792" alt="Screenshot 2023-01-19 at 3 47 39 PM" src="https://user-images.githubusercontent.com/1405144/213419707-31a12994-49c9-4bd9-860e-e176985aa09c.png">


## Checklist
- I have considered whether it makes sense to add tests for my changes
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...
